wordpress-mobile/WordPress-iOS
# ULS Get Started: show Email Address field

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/404
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/416

This updates Auth to show the Email Address field in the Get Started view.

To test:
- Enable the `unifiedWordPress` Feature Flag.
- Log In > Continue with WordPress.com.
- Verify the Email Address field is displayed.

| ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-08-26 at 16 30 48](https://user-images.githubusercontent.com/1816888/91363280-88e69400-e7b9-11ea-9ab8-9187b041d6ac.png) | ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-08-26 at 16 30 51](https://user-images.githubusercontent.com/1816888/91363291-8f750b80-e7b9-11ea-980d-822bd29b6ef4.png) |
|--------|-------|


PR submission checklist:
